src/features/expense/ui/DayExpenseList/DayExpenseList.tsx

```
@@ -0,0 +1,18 @@
+import { useExpenseStore } from '@/features/expense/model/expenseStore';
+
+const DayExpenseList = () => {
+  const month = useExpenseStore((state) => state.month);
+  const expenses = useExpenseStore((state) => state.expenses);
+
+  // const expenseList = ;
+
+  return (
+    <div className='bg-gray-100'>
+      {expenses.map(() => (
+        <DayExpenseItem />
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)